doc/users/next_whats_new/mouse_rotation.rst

```diff
@@ -4,9 +4,12 @@ Rotating 3d plots with the mouse
 Rotating three-dimensional plots with the mouse has been made more intuitive.
 The plot now reacts the same way to mouse movement, independent of the
 particular orientation at hand; and it is possible to control all 3 rotational
-degrees of freedom (azimuth, elevation, and roll). It uses a variation on
-Ken Shoemake's ARCBALL [Shoemake1992]_.
+degrees of freedom (azimuth, elevation, and roll). By default,
+it uses a variation on Ken Shoemake's ARCBALL [1]_.
+The particular style of mouse rotation can be set via
+``rcParams.axes3d.mouserotationstyle``.
+See also :doc:`/api/toolkits/mplot3d/view_angles`.
```

Comment:
Possible to deep link to `#rotation-with-mouse`? Not sure

Reply:
> Possible to deep link to `#rotation-with-mouse`? Not sure

I added a label, and then ``:ref:`toolkit_mouse-rotation` `` appears to do the job.